packages/shared/src/components/cards/v1/FeedbackCard.tsx

```diff
@@ -10,7 +10,7 @@ import UpvoteIcon from '../../icons/Upvote';
 import { Post, UserPostVote } from '../../../graphql/posts';
 import { usePostFeedback } from '../../../hooks';
 import CloseButton from '../../CloseButton';
-import { PostCardFooter } from './PostCardFooter';
+// import { PostCardFooter } from './PostCardFooter';
```

Comment:
Removed, or another ticket?
If so please mention ticket TODO

Reply:
good catch, forgot to remove these